docker/README.md

```diff
@@ -0,0 +1,127 @@
+
+# Dockerized PiKaraoke
+
+PiKaraoke is a "KTV"-style karaoke song search and queueing system, originally designed to work on Raspberry Pi, OSX, Windows, and Linux. This repository contains a Dockerized version of PiKaraoke, built on an alpine base, making it easy to set up and run on any system with Docker support.
+
+The original project can be found <a href="https://github.com/vicwomg/pikaraoke">here</a>
+
+## Features
+
+The Dockerized version retains all the features of the original PiKaraoke, including:
+
+- Web interface for multiple users to queue tracks
+- Player/splash screen with connection QR code and "Next up" display
+- Searching/browsing a local song library
+- Adding new songs from YouTube
+- mp3 + cdg support, including compressed .zip bundles
+- Pause/Skip/Restart and volume control
+- Advanced editing of downloaded file names
+- Queue management
+- Key Change / Pitch shifting
+- Lock down features with admin mode
+
+(Refer to the original PiKaraoke features for more details.)
+
+---
+
+## Support the Original Project
+
+If you want to support the original PiKaraoke project with a monetary tip, it's much appreciated by the original developer:
+
+[![Buy Me A Coffee](https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png)](https://www.buymeacoffee.com/vicwomg)
+
+---
+
+
+## Running Dockerized PiKaraoke
+
+### Prerequisites
+
+- Docker and Docker Compose installed on your system
+
+### Installation and Launch
+
+
+## Using Docker Compose Command Line
+
+1. **Clone the Repository**:
+   First, clone the repository from GitHub to get the `docker-compose.yml` file and any other necessary configurations:
+   ```bash
+   git clone https://github.com/honestlai/pikaraoke-docker.git
```

Comment:
Would this URL actually be https://github.com/vicwomg/pikaraoke.git now?

Reply:
Similarly line 51 should be `cd pikaraoke/docker` no?

Reply:
Nice catch... I was too tired to update the readme last night. There's actually quite of a bit I'm going to add/change that should make deployment options more clear.